src/Symfony/Component/Messenger/Transport/AmqpExt/AmqpRoutingKeyStamp.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\AmqpExt;
+
+use Symfony\Component\Messenger\Stamp\StampInterface;
+
+/**
+ * @author Guillaume Gammelin <ggammelin@gmail.com>
+ *
+ * @experimental in 4.3
+ */
+final class AmqpRoutingKeyStamp implements StampInterface
+{
+    /**
+     * @var string
+     */
+    private $routingKey;
```

Comment:
phpdoc isn't needed here (it's redundant)

Reply:
@G15N you resolved the comment but didn't remove the PhpDoc 🤔

Reply:
Yup, I haven't push the commit yet, my bad.